programs/uxd/src/instructions/credix_lp/collect_profit_of_credix_lp_depository.rs

```diff
@@ -467,6 +465,11 @@ impl<'info> CollectProfitOfCredixLpDepository<'info> {
 impl<'info> CollectProfitOfCredixLpDepository<'info> {
     pub(crate) fn validate(&self) -> Result<()> {
         validate_is_program_frozen(self.controller.load()?)?;
+        require!(
+            self.depository.load()?.profits_beneficiary_collateral
+                != Pubkey::new_from_array([0u8; 32]),
+            UxdError::UninitializedProfitsBeneficiaryCollateral
```

Comment:
I was thinking this might not be necessary since the accounts constraint: token::mint check should already make sure this is a valid token account

Reply:
sounds good king, will remove then